From c6759b471aa597920060f81052a8459fea0354dc Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:08:37 -0400
Subject: [tools] perf/documentation: Add description for conditional branch filter

Message-id: <1410181752-25631-273-git-send-email-jolsa@redhat.com>
Patchwork-id: 91844
O-Subject: [PATCH RHEL7.1 BZ1134356 272/307] perf/documentation: Add description for conditional branch filter
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 3e39db4ae2a92ae9e338e8066411b694b0edcb31
Author: Anshuman Khandual <khandual@linux.vnet.ibm.com>
Date: Thu May 22 12:50:10 2014 +0530
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Documentation/perf-record.txt b/tools/perf/Documentation/perf-record.txt
index c71b0f3..d460049 100644
--- a/tools/perf/Documentation/perf-record.txt
+++ b/tools/perf/Documentation/perf-record.txt
@@ -184,9 +184,10 @@ following filters are defined:
  - in_tx: only when the target is in a hardware transaction
  - no_tx: only when the target is not in a hardware transaction
  - abort_tx: only when the target is a hardware transaction abort
+ - cond: conditional branches
 
 +
-The option requires at least one branch type among any, any_call, any_ret, ind_call.
+The option requires at least one branch type among any, any_call, any_ret, ind_call, cond.
 The privilege levels may be omitted, in which case, the privilege levels of the associated
 event are applied to the branch filter. Both kernel (k) and hypervisor (hv) privilege
 levels are subject to permissions.  When sampling on multiple events, branch stack sampling
-- 
1.7.1